libffi-rs/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "MIT/Apache-2.0"
 keywords = ["ffi", "libffi", "closure", "c"]
 categories = ["development-tools::ffi"]
 edition = "2018"
+rust-version = "1.78"
```

Comment:
Could this be set at a workspace level instead?

Reply:
Sure, would you like all the other redundant fields also moved up to workspace?

Reply:
Sure, that's probably for the best.

Reply:
I promoted the fields that look stable.